packages/multichain-network-controller/src/types.ts

```diff
@@ -55,21 +47,21 @@ export type NonEvmNetworkConfiguration = CommonNetworkConfiguration & {
 
 // TODO: The controller only supports non-EVM network configurations at the moment
 // Once we support Caip chain IDs for EVM networks, we can re-enable EVM network configurations
-// export type EvmNetworkConfiguration = CommonNetworkConfiguration & {
-//   isEvm: true;
-//   /**
-//    * The RPC endpoints of the network.
-//    */
-//   rpcEndpoints: string[];
-//   /**
-//    * The index of the default RPC endpoint.
-//    */
-//   defaultRpcEndpointIndex: number;
-// };
+export type EvmNetworkConfiguration = CommonNetworkConfiguration & {
```

Comment:
I need to add this back since it's required at least in extension tp combine the network list from both controllers